CONTRIBUTING.md

```diff
@@ -35,9 +35,7 @@ behavior that is not occurring.
 We use [Scalafmt](https://scalameta.org/scalafmt/) to format our code! Please format your code before opening a Pull Request.
 
 ### Tests aren’t optional
```

Comment:
I would merge the two test sections below into a single section titled "Running and writing tests":

````markdown
### Running and writing tests

**Important**: Please include tests with any code contributions

Writing tests before the implementation is strongly encouraged. 

To run tests:
```sh
$ sbt test
$ sbt clientJVM/testOnly // to run tests only for the JVM platform
$ sbt clientJS/testOnly // to run tests only for the JS platform
```

Code contributions must comply with a minimum of 80% coverage rate. 

To run a coverage report of the whole project:
```sh
sbt coverage test coverageReport
```

````

Reply:
Done in 353dc34d774814dc57b15fe6b4f5499514c9e805